val/common/sys_arch_src/gic/its/acs_gic_its.c

```diff
@@ -251,6 +353,23 @@ WriteCmdQMAPI(
     g_cwriter_ptr[its_index] = g_cwriter_ptr[its_index] + ITS_NEXT_CMD_PTR;
 }
 
+static void
+WriteCmdQMAPTI(
```

Comment:
Please remove WriteCmdQMAPI as this api will not be used.

Reply:
We will check and remove if not used.

Reply:
WriteCmdQMAPI is from orginal arm release, we have added WriteCmdQMAPTI which is being used.